test/e2e/commands/validate_data/test_generic_violations.py

```diff
@@ -139,6 +142,8 @@ def test_reformat_cardinality_violation(cardinality_violation: ValidationReportG
     for one_result, expected_info in zip(alphabetically_sorted, expected_info_tuples):
         assert one_result.res_id == expected_info[0]
         assert one_result.problem_type == expected_info[1]
+    assert not _get_validation_status(sorted_problems, IS_ON_PROD_SERVER)
+    assert not _get_validation_status(sorted_problems, IS_NOT_ON_PROD_SERVER)
```

Comment:
Nitpick: Probably a matter of taste, but I think that IS_ON_PROD_SERVER and IS_NOT_ON_PROD_SERVER are less handy than `is_on_prod_server=True/False`